README.md

```diff
@@ -71,605 +137,84 @@ SYNOPSIS
-                                    --db_cpus=[1,4,8,16] --run_db_as_batch=False \
-                                    --net_ifname=ipogif0 --device=GPU
-```
-
-This option is recommended as it's easy to launch in interactive allocations and
-as a batch submission, but if you need to specify separate hosts for the database
-you can look into the following two methods.
-
-A batch submission for this first option would look like the following for Slurm
-based systems.
-
-```bash
-#!/bin/bash
-
-#SBATCH -N 136
-#SBATCH --exclusive
-#SBATCH -t 10:00:00
-
-python driver.py inference_standard --client_nodes=[20,40,60,80,100,120] \
-                                    --db_nodes=[4,8,16] --db_tpq=[1,2,4] \
-                                    --db_cpus=[1,4,8,16] --run_db_as_batch=False
-                                    --net_ifname=ipogif0 --device=CPU
-```
-
-2. Same as 1, but specify hosts for the database
-```bash
-# alloc must contain at least 120 (max client_nodes) + 16 nodes (max db_nodes)
-# db nodes must be fixed if hostlist is specified
-python driver.py inference_standard --client_nodes=[20,40,60,80,100,120] \
-                                    --db_nodes=[16] --db_tpq=[1,2,4] \
-                                    --db_cpus=[1,4,8,16] --db_hosts=[nid0001, ...] \
-                                    --net_ifname=ipogif0 --device=CPU
-
-```
-
-3. Launch database as a separate batch submission each time
-```bash
-# must obtain separate allocation for client driver through interactive or batch submission
-# if batch submission, compute nodes must have access to slurm
-python driver.py inference_standard --client_nodes=[20,40,60,80,100,120] \
-                                    --db_nodes=[4,8,16] --db_tpq=[1,2,4] \
-                                    --db_cpus=[1,4,8,16] --batch_args='{"C":"V100", "exclusive": None}' \
-                                    --net_ifname=ipogif0 --device=GPU
-```
-
-All three options will conduct ``n`` scaling tests where ``n`` is the multiple of
-all lists specified as options.
+## Results
 
-### Throughput
+The output organization of the performance results is detail below.
 
-The throughput tests run as an MPI program where a single SmartRedis C++ client
-is initialized on every rank.
+### Results File Structure
 
-Each client performs 10 executions of the following commands
+When a scalability test is first initialized, a nested folder named `results/'exp_name'`
+is created. The `exp_name` is captured by the `exp_name` flag value when you run your 
+scaling test. Such that if I ran `python driver.py inference_standard` with the default
+`exp_name`, the path to my results would be `results/inference-standard-scaling`. 
 
-  1) ``put_tensor``     (send image to database)
-  2) ``unpack_tensor``  (Retrieve the inference result)
+Each time you run a scalability test it is considered a single run. This is how the 
+`results/'exp_name'` is organized. Per time you run a scalability test, the results 
```

Comment:
"Per time you run a scalability test" -> Each time the scaling test is run, a new folder will be created with the time of execution